package.json

```
@@ -35,6 +35,7 @@
     "@babel/preset-typescript": "7.26.0",
     "@patternfly/react-icons": "5.4.2",
     "@playwright/test": "1.50.1",
+    "@currents/playwright": "^1.9.4",
```

Can you pin the version, dependabot likes pinned version more

```suggestion
    "@currents/playwright": "1.9.4",
```

---

Fixed!